src/BenchmarksApps/TodosApi/Program.cs

```diff
@@ -0,0 +1,76 @@
+using Microsoft.Extensions.Logging.Configuration;
+using Npgsql;
+using TodosApi;
+
+var builder = WebApplication.CreateSlimBuilder(args);
+
+// Load custom configuration
+var settingsFiles = new[] { "appsettings.json", $"appsettings.{builder.Environment.EnvironmentName}.json" };
+foreach (var settingsFile in settingsFiles)
+{
+    builder.Configuration.AddJsonFile(builder.Environment.ContentRootFileProvider, settingsFile, optional: true, reloadOnChange: true);
+}
+
+#if DEBUG || DEBUG_DATABASE
+builder.Configuration.AddUserSecrets<Program>();
+#endif
+
+#if ENABLE_LOGGING
+// Configure logging
+builder.Logging
+    .AddConfiguration(builder.Configuration.GetSection("Logging"))
+    .AddSimpleConsole();
+#endif
+
+// Configure authentication & authorization
+builder.Services.AddAuthentication()
+    .AddJwtBearer(JwtConfiguration.ConfigureJwtBearer(builder));
+
+builder.Services.AddAuthorization();
+
+// Configure data access
+var connectionString = builder.Configuration.GetConnectionString("TodoDb")
+    ?? builder.Configuration["CONNECTION_STRING"]
```

Comment:
Why do we check this one? But the exception only mentions `ConnectionStrings:TodoDb`

Reply:
Yeah that's a carry over from TrimmedTodo. We can just simplify and go with the one key.